sae_lens/training/activations_store.py

```diff
@@ -700,16 +712,13 @@ def next_batch(self):
             self._dataloader = self.get_data_loader()
             return next(self.dataloader)
 
-    def state_dict(self) -> dict[str, torch.Tensor]:
-        result = {
-            "n_dataset_processed": torch.tensor(self.n_dataset_processed),
+    def save_state(self, path: Path):
+        state = {
+            "norm_scaling_factor": self.estimated_norm_scaling_factor,
         }
-        if self._storage_buffer is not None:  # first time might be None
-            result["storage_buffer"] = self._storage_buffer
-        return result
 
-    def save(self, file_path: str):
-        save_file(self.state_dict(), file_path)
```

Comment:
Don't read this as a rewrite of the removed functions, it's an unrelated removal and addition. The removed ones were unused as far as I can tell, but if they're a desired public interface I'll put them back in!